guides/common/modules/proc_upgrading-project-from-el7-to-el8.adoc

```diff
@@ -1,15 +1,47 @@
-[id="Upgrading-Project-from-EL7-to-EL8_{context}"]
+[id="Upgrading-Project-from-Enterprise-Linux-7-to-Enterprise-Linux-8_{context}"]
+ifdef::foreman-el,katello[]
 = Upgrading {Project} from Enterprise Linux 7 to Enterprise Linux 8
+endif::[]
+ifdef::satellite[]
+= Upgrading {Project} from {RHEL} 7 to {RHEL} 8
+endif::[]
 
+ifdef::foreman-el,katello[]
 Use this procedure to upgrade your {Project} installation from Enterprise Linux 7 to Enterprise Linux 8.
 LEAPP is the supported tool for Enterprise Linux 7 to Enterprise Linux 8 upgrades.
+endif::[]
+
+ifdef::satellite[]
+Use this procedure to upgrade your {Project} installation from {RHEL} 7 to {RHEL} 8.
+endif::[]
 
+ifdef::foreman-el,katello[]
 .Prerequisites
 * {Project} {ProjectVersion} running on Enterprise Linux 7.
 * Access to available repositories or a local mirror of repositories.
-* No outstanding updates.
 * {Project} installations running on CentOS 7 can be upgraded to CentOS Stream 8 or a {RHEL} rebuild.
 * {Project} installations running on {RHEL} 7 can be upgraded to {RHEL} 8.
+endif::[]
+
+ifdef::satellite[]
+.Prerequisites
+* {Project} {ProjectVersion} running on {RHEL} 7.
+* Access to available repositories or a local mirror of repositories.
+* {Project} installations running on {RHEL} 7 can be upgraded to {RHEL} 8.
+
+.Backup and Restore
+. You can create a {RHEL} 8 system with identical specifications and the same hostname as the {RHEL} 7 system using the fresh install migration method.
+. This method requires a considerable amount of system downtime while the system backup is restored on the {RHEL} 8 system.
+. To reduce the amount of downtime, you can continue to use the {RHEL} 7 system on the target machine while the backup is being restored.
+. Any changes you make during this time are not included in the backup on the {RHEL} 8 system on the source machine.
+. Storage space is required for the backup and access to a second machine to host the new {RHEL} 8 system.
+
+.In-Place Migration Using LEAPP
+. The in-place migration method requires several hours less downtime than the fresh install method.
+. You do not need to provide storage space for a backup or a second machine for {RHEL} 8, as the migration can occur in place on the {RHEL} 7 machine.
+. Downtime cannot be avoided entirely by using the {RHEL} 7 system while the {RHEL} 8 system is preparing because the in-place migration occurs on the same machine.
+. The in-place migration using LEAPP is not possible with FIPS-enabled systems.
```

Comment:
I still think that having these two sections in here right now completely breaks the flow of the document.

See https://github.com/theforeman/foreman-documentation/pull/1314#pullrequestreview-1003980603 for a longer explanation of my thoughts.

Reply:
@evgeni I defer to your expertise on this and I will remove these sections from the document. :-)